sourcegraph/README.md

```diff
@@ -0,0 +1,75 @@
+# Sourcegraph Datadog Services Map Extension
+
+## Overview
+
+![Datadog Services Map Screenshot](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sourcegraph/images/sourcegraph-datadog-services-map-extension.png)
```

Comment:
Not sure if this is also the case for links to images, but I the doc team recommends to put the links at the end of the readme file

Reply:
yes, but we don't typically do it with images